query/metaquery/handlers.go

```diff
@@ -251,6 +254,9 @@ func inspect(input *HandlerInput) error {
 		if !schemaFound {
 			searchPath := input.Schema.SearchPath
 
+			// add the temporary schema to the search_path so that it becomes searchable
+			searchPath = append(searchPath, input.Schema.TemporarySchemaName)
```

Comment:
why do we need this?

Reply:
The iteration happens through the current search path - but the temporary schema is not a part of that list. Need to add it to the end.